node/src/components/storage.rs

```diff
@@ -194,7 +202,10 @@ pub struct Storage {
     /// The state storage database.
     #[data_size(skip)]
     state_store_db: Database,
-    /// The finalized approvals database.
+    /// The finalized deploy approvals database.
+    #[data_size(skip)]
+    finalized_deploy_approvals_db: Database,
+    /// The finalized transaction approvals database.
     #[data_size(skip)]
     finalized_approvals_db: Database,
```

Comment:
```suggestion
    finalized_transaction_approvals_db: Database,
```

Reply:
Fixed in [66ceea6](https://github.com/casper-network/casper-node/pull/4236/commits/66ceea68545c848ce0bcfa7a2fbda6aa1dd769c4).